pkg/util/mount/nsenter_mount.go

```diff
@@ -39,14 +40,31 @@ import (
 //     performed in the host's mount namespace do not propagate out to the
 //     bind-mount in this docker version.
 // 2.  The host's root filesystem must be available at /rootfs
-// 3.  The nsenter binary must be at /nsenter in the container's filesystem.
+// 3.  The nsenter binary must be on the PATH in the container's filesystem.
 // 4.  The Kubelet process must have CAP_SYS_ADMIN (required by nsenter); at
 //     the present, this effectively means that the kubelet is running in a
 //     privileged container.
+// 5.  The volume path used by the Kubelet must be the same inside and outside
+//     the container and be writable by the container (to initialize volume)
+//     contents. TODO: remove this requirement.
```

Comment:
Should we document the fact that the container image must contain mount, umount, and findmnt, or the rootfs you mount inside the container does?